### DISCUSSION TITLE
Is there an easy way to install the capacitor 5-compatible versions of the plugins before the official release?

### DISCUSSION THREAD

**Discussion by ptmkenny** [capawesome-team/capacitor-plugins#30] [Q&A] [answered]
To test Capacitor 5 prior to release, we can switch the version to `next` in `package.json`.

I see that [Capacitor 5 compatibility](https://github.com/capawesome-team/capacitor-plugins/issues/2) has already been added to this set of plugins, too.

However, I don't see a way that I can update `@capawesome/capacitor-badge` to a beta version that includes the Capacitor 5 support.  How can I do this?

I checked the "Releases" section for this repo but it's empty, so I don't know how to get the release info to see what's available.

**Comment by robingenz** [comment, marked as answer]
Thank you for your request.
There is no dist tag yet. But you can install the dev version, see https://github.com/capawesome-team/capacitor-plugins/pull/20#pullrequestreview-1376458030.

**Reply by ptmkenny** [reply]
Awesome, thank you!
